lib/block-supports/layout.php

```diff
@@ -816,7 +816,7 @@ function gutenberg_render_layout_support_flag( $block_content, $block ) {
 			break;
 		}
 
-		if ( false !== strpos( $processor->get_attribute( 'class' ), $inner_block_wrapper_classes ) ) {
+		if ( false !== strpos( $processor->get_attribute( 'class' ) ?? '', $inner_block_wrapper_classes ) ) {
```

Comment:
we might want to trap the value of the `$class` attribute here and use `is_string()` instead of `?? ''` because a `class` attribute could also be boolean and thus return `true` here.

```php
$class_attribute = $processor->get_attribute( 'class' );
if ( is_string( $class_attribute ) && false !== strpos( $class_attribute, $inner_block_wrapper_classes ) ) {
	break;
}
```

good find here; thanks for the fix!